src/Hydration.js

```diff
@@ -11,15 +11,30 @@ class Hydration {
     );
   }
 
+  // We are not using this function and this has not been tested!!!
```

Comment:
This seems to be the function that automatically loaded the last 7 days of data so that the user wouldn't have to input today as the date. Really great functionality